Yoast/wordpress-seo
# Fix metabox from overflowing onto sidebar

## Summary

This PR can be summarized in the following changelog entry:
* Fixes a bug where the metabox would overflow onto the sidebar preventing users from changing the categories or using other metaboxes in the sidebar.

## Relevant technical choices:

* I put the `overflow: auto` on the .inside div because the child divs are styled with `display: table-cell` which makes adding max-width hard.

## Test instructions

This PR can be tested by following these steps:

* Go to the post edit screen.
* Make your screen smaller (or your browser window).
* Observe that you can now edit your categories.
